src/utils/util.h

```diff
@@ -10,6 +10,7 @@ extern char *test_config_file;
 void check_memory_allocation(void *check_me);
 void replace_in_string(char *to_replace, char replace_this, char replace_with);
 void show_help();
-void check_if_running();
+int is_running();
```

Comment:
Check if possible to add a test. Could be that locally if the process is running will give some headache though.